project/src/pages/main-page/main-page.tsx

```diff
@@ -23,23 +25,31 @@ function MainPage({rooms}: MainPageProps): JSX.Element {
   return (
     <div className="page page--gray page--main">
       <Header/>
-      <main className="page__main page__main--index">
+      <main className={cn('page__main', 'page__main--index', {
+        'page__main--index-empty': rooms.length === 0
```

Comment:
Еще варианты, выбирай какой нравится
```suggestion
        'page__main--index-empty': !!rooms.length,
        'page__main--index-empty': !Boolean(rooms.length)
```